pages/FAQ/_index.md

```diff
@@ -254,47 +242,44 @@ killall xdg-desktop-portal
 sleep 4
 /usr/lib/xdg-desktop-portal &
 ```
-launched with `exec-once` should fix all issues. Adjust the sleep durations to taste.
-
-### My cursor in QT apps is too big!
 
-The QT cursor size can be fixed by setting the envvar `XCURSOR_SIZE=24`. You
-should probably add it to your exported envvars.
+launched with `exec-once` should fix all issues. Adjust the sleep durations to taste.
 
 ### How do I export envvars for Hyprland?
 
 As with any Display Server, Xorg included, you should probably make a script to
 launch it, for example:
 
-```ini
+```bash
 export AMONG_US=1
 exec Hyprland
 ```
 
 and launch that.
 
 For Display Manager users, you can replace the `exec` entry in
-the `.desktop` file to point to your script.
+the `.desktop` file to point to your script. You are recommended
+to use absolute paths, such as `/home/username/Script` instead of `~/Script`
 
-### I get random white flashes!
+### I get random white flashes
 
 Try disabling VFR with `misc:no_vfr=1`.
 
 ### How do I make Hyprland draw as little power as possible on my laptop?
 
-I assume you already have `damage_tracking` on full. If you don't, do it. It's
+I assume you already have `damage_tracking` on full. If you don't, change it. It's
 heavily advised to use `full` regardless of anything.
 
-Optimization options:
-
-*feel free to ignore any that you find causing issues*
+#### Useful Optimizations
 
-`decoration:blur_new_optimizations = true`
+* `decoration:blur_new_optimizations = true`, to use new optimizations for
+   blurring.
 
-`decoration:blur = false`
+* `decoration:blur = false` and `decoration:drop_shadow = false` to disable
+   fancy but battery hungry effects.
 
-`decoration:drop_shadow  = false`
+* `misc:no_vfr = false`, since it has been proven that it consumes less power.
```

Comment:
yeah no shit. `will lower your display's refresh rate when nothing is happening on-screen.`